### ISSUE TITLE
Stuck at loading in all examples but 5, 6 and 7

### ISSUE THREAD

**Issue by jmcarcell** [GibbsConsulting/django-plotly-dash#388]
As the title says, I try to run but instead of the examples I see a box with 'Loading ...' that never changes. There are no errors on the server side. I'm on Arch Linux with python 3.10 but I have also tried in Centos 8 and I get the same result, maybe I'm doing something wrong? This is what I've been running (the redis step is already done):
```
git clone https://github.com/GibbsConsulting/django-plotly-dash
cd django-plotly-dash
vim make_env # Change the python version, in this case to 3.10
vim requirements.txt # Add a ',<4.0.0' for Django because there are errros when using Django 4.x.x
./make_env
./prepare_demo
```
When I click on the first example I get on the server:
```
HTTP GET /demo-one 200 [0.03, 127.0.0.1:34912]
HTTP GET /static/demo/demo.css 304 [0.01, 127.0.0.1:34912]
HTTP GET /static/demo/logo.svg 304 [0.01, 127.0.0.1:34910]
HTTP GET /django_plotly_dash/app/SimpleExample/ 200 [0.01, 127.0.0.1:34910]
```
And this is the result:
![2022-03-14-171132_3815x1664_scrot](https://user-images.githubusercontent.com/22276694/158214119-436fa9f9-9fbe-4dff-88e9-4c5eff3ba980.png)
Any ideas? I have tried different browsers, I have seen #360 but it hasn't helped.

**Comment by GibbsConsulting** [comment]
Can you open up a developer tools console in your browser and look for error messages (and also any network requests that are failing)?

What version of Dash (and other packages) are you running? `pip freeze` will report this.

Note that you'll want to use the `prepare_redis` script for all of the examples to work, but I don't think that is the main issue here.

**Comment by jmcarcell** [comment]
On the console I only get one error about `favicon.ico` not being found on Firefox, but that happens on all the examples whether they work or not. Google Chrome reports 
```
Blocked report execution in '<URL>' because the document's frame is sandboxed and the 'allow-scripts' permission is not set.
```
This is reported only on the examples that do not work so it's looking suspicious. Maybe something like #379 is needed? I will try add that permission.
This is the list of packages that I get in the venv after `./make_env`:
```
aioredis==1.3.1
alabaster==0.7.12
asgiref==3.5.0
astroid==2.9.3
async-timeout==4.0.2
attrs==19.3.0
autobahn==22.2.2
Automat==20.2.0
Babel==2.9.1
beautifulsoup4==4.10.0
bleach==4.1.0
Brotli==1.0.9
certifi==2021.10.8
cffi==1.15.0
channels==2.4.0
channels-redis==3.4.0
charset-normalizer==2.0.12
click==8.0.4
colorama==0.4.4
constantly==15.1.0
coverage==6.3.2
coveralls==3.3.1
cryptography==36.0.1
daphne==2.5.0
dash==1.20.0
dash-bootstrap-components==0.13.1
dash-core-components==1.16.0
dash-html-components==1.1.3
dash-renderer==1.9.1
dash-table==4.11.3
Deprecated==1.2.13
Django==3.2.12
django-bootstrap4==22.1
-e git+https://github.com/GibbsConsulting/django-plotly-dash@a0f86e53059391a2362c0c9737fe8db160ffcd0b#egg=django_plotly_dash
django-redis==5.2.0
docopt==0.6.2
docutils==0.17.1
dpd-components==0.1.0
dpd-static-support==0.0.5
Flask==2.0.3
Flask-Compress==1.11
future==0.18.2
grip==4.6.0
hiredis==2.0.0
hyperlink==21.0.0
idna==3.3
imagesize==1.3.0
importlib-metadata==4.11.3
incremental==21.3.0
iniconfig==1.1.1
isort==5.10.1
itsdangerous==2.1.1
jeepney==0.7.1
Jinja2==3.0.3
keyring==23.5.0
lazy-object-proxy==1.7.1
livereload==2.6.3
Markdown==3.3.6
MarkupSafe==2.1.0
mccabe==0.6.1
msgpack==1.0.3
numpy==1.22.3
packaging==21.3
pandas==1.4.1
path-and-address==2.0.1
pkginfo==1.8.2
platformdirs==2.5.1
plotly==5.6.0
pluggy==1.0.0
py==1.11.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycparser==2.21
Pygments==2.11.2
pylint==2.12.2
pyOpenSSL==22.0.0
pyparsing==3.0.7
pytest==7.1.0
pytest-cov==3.0.0
pytest-django==4.5.2
python-coveralls==2.9.3
python-dateutil==2.8.2
pytz==2021.3
PyYAML==6.0
readme-renderer==34.0
redis==4.1.4
requests==2.27.1
requests-toolbelt==0.9.1
rfc3986==2.0.0
SecretStorage==3.3.1
service-identity==21.1.0
six==1.16.0
snowballstemmer==2.2.0
soupsieve==2.3.1
Sphinx==4.4.0
sphinx-autobuild==2021.3.14
sphinx-rtd-theme==1.0.0
sphinxcontrib-applehelp==1.0.2
sphinxcontrib-devhelp==1.0.2
sphinxcontrib-htmlhelp==2.0.0
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.3
sphinxcontrib-serializinghtml==1.1.5
sqlparse==0.4.2
tenacity==8.0.1
toml==0.10.2
tomli==2.0.1
tornado==6.1
tqdm==4.63.0
twine==3.8.0
Twisted==22.2.0
txaio==22.2.1
typing_extensions==4.1.1
urllib3==1.26.8
webencodings==0.5.1
Werkzeug==2.0.3
whitenoise==6.0.0
wrapt==1.13.3
zipp==3.7.0
zope.interface==5.4.0
```

**Comment by jmcarcell** [comment]
I can report some success when changing `django-plotly-dash/django_plotly_dash/templates/django_plotly_dash/plotly_app.htlm` to:
```
<div style="{{dstyle}}">
  <iframe src="{{app.base_url}}" style="{{istyle}}" frameborder="{{fbs}}" sandbox="allow-downloads allow-scripts allow-same-origin"></iframe>
</div>
```
Now most of the examples work although there are a couple that are not displayed correctly, like number 3, but I think that's a different issue and there aren't any errors this time:

![2022-03-15-084659_2881x1863_scrot](https://user-images.githubusercontent.com/22276694/158330398-9648f06a-1983-4ec6-aaa3-d5990e088475.png)

I'll play with this a bit more and make a PR with the changes

**Comment by GibbsConsulting** [comment]
Other than using Django 4.0, this should be resolved by the release of v2.0.0

Django 4.n use is tracked as #403

Closing this issue, but @jmcarcell please reopen (or preferably, create a new issue) if there is anything outstanding.
